image/templates/helm/stackrox-central/templates/02-config-controller-01-rbac.yaml

```diff
@@ -0,0 +1,44 @@
+{{- include "srox.init" . -}}
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Should we make it a `Role` instead of a `ClusterRole`?  I'm kind of thinking that this helm chart shouldn't install any cluster-scoped resources.

Reply:
I was about to ask this one: Why is this ClusterRole and wouldn't a role suffice?